hdwallet/cryptocurrencies.py

```diff
@@ -3213,18 +3213,18 @@ class LitecoinMainnet(Cryptocurrency):
     })
 
     EXTENDED_PRIVATE_KEY = ExtendedPrivateKey({
-        "P2PKH": 0x019d9cfe,
-        "P2SH": 0x019d9cfe,
-        "P2WPKH": 0x04b2430c,
-        "P2WPKH_IN_P2SH": 0x01b26792,
+        "P2PKH": 0x488ade4,
+        "P2SH": 0x488ade4,
+        "P2WPKH": None,
+        "P2WPKH_IN_P2SH": None,
         "P2WSH": None,
         "P2WSH_IN_P2SH": None
     })
     EXTENDED_PUBLIC_KEY = ExtendedPublicKey({
-        "P2PKH": 0x019da462,
-        "P2SH": 0x019da462,
-        "P2WPKH": 0x04b24746,
-        "P2WPKH_IN_P2SH": 0x01b26ef6,
+        "P2PKH": 0x488b21e,
+        "P2SH": 0x488b21e,
```

Comment:
It works in both P2PKH and P2SH but the standard of Litecoin EPrivateKey is ``0x019d9cfe`` and  EPublicKey is ``0x019da462``.

Reply:
Use this on LitecoinMainnet

```python3
    EXTENDED_PRIVATE_KEY = ExtendedPrivateKey({
        "P2PKH": 0x019d9cfe,  # 0x488ade4
        "P2SH": 0x019d9cfe,  # 0x488ade4
        "P2WPKH": None,
        "P2WPKH_IN_P2SH": None,
        "P2WSH": None,
        "P2WSH_IN_P2SH": None
    })
    EXTENDED_PUBLIC_KEY = ExtendedPublicKey({
        "P2PKH": 0x019da462,  # 0x488b21e
        "P2SH": 0x019da462,  # 0x488b21e
        "P2WPKH": None,
        "P2WPKH_IN_P2SH": None,
        "P2WSH": None,
        "P2WSH_IN_P2SH": None
    })
```

Reply:
Got the data by the ltc chainparams.
https://github.com/litecoin-project/litecoin/blob/master/src/chainparams.cpp#L132

Reply:
Okay, Maybe I will include option to access both extends keys next time. Good Job :)